# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../uclibc/patches/procinfo-uclibc.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- procinfo-18/routines.c.orig	2005-01-27 00:30:34.000000000 -0800
+++ procinfo-18/routines.c	2005-01-27 00:32:21.000000000 -0800
@@ -146,7 +146,7 @@
 	exit (0);
     } else {
 	printf ("%s%s%s", ve, se, tgoto (cm, 0, li));
-	printf ("[%s]\n", sys_siglist[i]);
+	printf ("[%s]\n", strsignal(i]));
 	exit (128 + i);
     }
 }
